doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -104,15 +104,46 @@ Neutron is an OpenStack project that provides "network connectivity as a service
+OpenStack Neutron supports open APIs and a pluggable backend where different plugins can be incorporated in the neutron-server.
 
+Plugins for various SDN controllers include either the standard ML-2 plugin or specific monolithic plugins.
 Neutron supports both core plugins that deal with L2 connectivity and IP address management, and service plugins that support services such as L3 routing, Load Balancers, Firewalls, etc.
 
+Below we will explore an example of SDN controller under LFN projects , that can be used integrated with Neutron plugin which it help to overcome a lot of challenges in Vanilla Neutron and provide many features that can be consumed by VNF/CNF 
+
+
+#### 3.2.5.1. Tungsten Fabric (SDN Controller)
+Tungsten Fabric, an open source SDN in Linux Foundation Networking (https://tungsten.io/), offers neutron networking through ML2 based pugin, additionally it supports advanced networking features beyond basic neutron networking via monolithic plugin. It also supports the same advanced networking features via CNI plugin in Kubernetes. Hence, it works as a multi-stack SDN to support VMs, containers, and baremetal workloads. It provides separation of control plane functions and data plane functions with its two components:
+
+- Tungsten Fabric Controller– a set of software services that maintains a model of networks and network policies, typically running on several servers for high availability
+- Tungsten Fabric vRouter– installed in each host that runs workloads (virtual machines or containers), the vRouter performings packet forwarding and enforces network and security policies
+
+It is based on proven, standards-based networking technologies that today support the wide-area networks of the world’s major service providers, but repurposed to work with virtualized workloads and cloud automation in data centers that can range from large scale enterprise data centers to much smaller telco POPs. It provides many enhanced features over the native networking implementations of orchestrators, including:
+- Highly scalable, multi-tenant networking
+- Multi-tenant IP address management
+- DHCP, ARP proxies to avoid flooding into networks
+- Efficient edge replication for broadcast and multicast traffic
+- Local, per-tenant DNS resolution
+- Distributed firewall with access control lists
+- Application-based security policies
+- Distributed load balancing across hosts
+- Network address translation (1:1 floating IPs and distributed SNAT)
+- Service chaining with virtual network functions
+- Dual stack IPv4 and IPv6
+- BGP peering with gateway routers
+- BGP as a Service (BGPaaS) for distribution of routes between privately managed customer networks and service provider networks
+
+
+Based on Network Layering concept that introduced ealier , Tungsten Fabric Controller , can be considered as SDNu and SDNo
```

Comment:
Need reference to RM suggest reword:

Based on the network layering concepts introduced in the Reference Model ChXX (provide link), the Tungsten Fabric Controller performs functions of both the SDN underlay (SDNu) and overlay (SDNo) controllers.

Reply:
i will add it after network layering concepts merged